dev-php/pecl-memcache/pecl-memcache-4.0.5.2-r1.ebuild

```diff
@@ -9,55 +9,21 @@ PHP_EXT_NEEDED_USE="session(-)?"
 DOCS=( README example.php )
 HTML_DOCS=( memcache.php )
 
-USE_PHP="php7-4 php8-0 php8-1"
+USE_PHP="php8-0 php8-1"
 
 inherit php-ext-pecl-r3
 
-USE_PHP="php7-4"
-
 KEYWORDS="~amd64 ~hppa ~ppc64 ~x86"
 
 DESCRIPTION="PHP extension for using memcached"
 LICENSE="PHP-3"
 SLOT="7"
 IUSE="+session"
 
-DEPEND="php_targets_php7-4? ( sys-libs/zlib )"
 RDEPEND="${DEPEND}
 	php_targets_php8-0? ( dev-php/pecl-memcache:8[php_targets_php8-0(-)?] )
 	php_targets_php8-1? ( dev-php/pecl-memcache:8[php_targets_php8-1(-)?] )
 "
 
 # The test suite requires memcached to be running.
 RESTRICT='test'
-
-src_prepare() {
-	if use php_targets_php7-4 ; then
-		php-ext-source-r3_src_prepare
-	else
-		default
-	fi
-}
-
-src_configure() {
```

Comment:
I don't get what the deal was here -- before, if you had !php7-4, it wouldn't run src_configure at all. Now we're going to run php-ext-source-r3's?

Reply:
this one in particular is completely broken and I dropped it